pegasus/sites.v3/code.org/public/edit-csp-table/splat.haml

```diff
@@ -127,15 +156,23 @@ angular: recordsApp
     }
 
   }]);
-
+  function makeColumn(name) {
+    return {
+      id: name,
```

Comment:
Alternately you could just add a comment indicating why there are two fields which often have the same state, and some details about what each one is for.